include/igl/fast_winding_number.cpp

```diff
@@ -0,0 +1,371 @@
+#include "fast_winding_number.h"
+#include "build_octree.h"
+#include "knn_octree.h"
+#include "parallel_for.h"
+#include "copyleft/cgal/point_areas_and_normals.h"
```

Comment:
This is not legal. You can only include things at or above. Separate into two files: one in igl/ that takes area as a input and then one in igl/copyleft/cgal that computes areas inside.